### DISCUSSION TITLE
Unauthorized on the gitbook docs page

### DISCUSSION THREAD

**Discussion by anupam-meesho** [Netflix/dispatch#3296] [General]
Hey Team,
I am not able to access https://app.gitbook.com/s/-Lb8PtMc6pK_s8P1rpOr/dispatch/administration-guide/incident/plugins/configuring-slack#slash-commands. or any of the docs pages. Please help.
<img width="1713" alt="image" src="https://user-images.githubusercontent.com/103025860/234057470-27b52c00-45e0-48f3-844f-8c27f7448305.png">

**Comment by ghost** [comment]
Hi @anupam-meesho, the documentation site recently moved and you should be able to view it here: https://netflix.github.io/dispatch/

Could you point me to where you found the old link in case it needs to be updated?

Thank you!

**Reply by anupam-meesho** [reply]
I have found the old link on the Netflix medium page: https://netflixtechblog.com/introducing-dispatch-da4b8a2a8072
